lntest/itest/lnd_misc_test.go

```diff
@@ -314,12 +324,21 @@ func testListChannels(ht *lntemp.HarnessTest) {
 	)
 
 	// Get the ListChannel response for Bob.
-	bobChannel := ht.QueryChannelByChanPoint(bob, chanPoint)
+	bobChannel := ht.QueryChannelByChanPoint(bob, chanPoint, false)
 	require.Equal(ht, aliceChannel.ChannelPoint, bobChannel.ChannelPoint,
 		"Bob's channel point mismatched")
 
+	// Query the channel again, this time with node alias lookup
+	bobChannelWithAlias := ht.QueryChannelByChanPoint(
+		bob, chanPoint, true,
+	)
+	// Alice's node alias
```

Comment:
nit: add newline before comment. Though does this comment add any value other than visual grouping itself?

Reply:
Not really, I removed it.